MBProgressHUD.m

```diff
@@ -945,6 +990,11 @@ @implementation MBBarProgressView
 #pragma mark - Lifecycle
 
 - (id)init {
+    // Ensure that this is an accessibility element and set the trait to allow percentage completion to be accessible.
+    self.isAccessibilityElement = YES;
+    self.accessibilityLabel = @"Progress";
+    self.accessibilityTraits = UIAccessibilityTraitUpdatesFrequently;
```

Comment:
This is redundant as we're calling through to `initWithFrame:` below which does the same.